{{cookiecutter.project_repo}}/.github/workflows/pythonpackage.yml

```diff
@@ -9,21 +9,40 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
```

Comment:
Are we dropping support for 3.6? I notice it's not in our up-to-date packages ie `cvtoolkit`

Reply:
We should also consider py36 inclusion/exclusion in tox.ini in this case also

Reply:
Yes we can drop support for 3.6

Reply:
It's also worth adding support for 3.9

Reply:
Added